docs/man/docker-events.1.md

```diff
@@ -45,23 +45,23 @@ After running docker events a container 786d698004576 is started and stopped
 (The container name has been shortened in the output below):
```

Comment:
If you're going to change the container ID to long form, can you please remove the above?